.github/workflows/haskell-ci.yml

```diff
@@ -0,0 +1,66 @@
+name: Tests
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: CI
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        resolver:
+          [
+            "nightly-2023-05-15",
```

Comment:
Recently 9.4.5 has turned to lts (lts-21.3 is the latest). Also it might make sense to bump to 9.2.8 (lts-20.26)